staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -100,6 +100,10 @@ type Config struct {
 	// needs to process an incoming event.
 	IndexerFuncs storage.IndexerFuncs
 
+	// CacheIndexers is used to accelerate the list operations, go back to normal list
```

Comment:
... operations. Falls back to a regular list ...

Reply:
fix it!